### PR TITLE
Findable module

### PR DESCRIPTION
**Pull request by janegreene**
Added find team id to a module for use in season repo and league repo 
tests are passing

### CODE REVIEW

File: lib/season_repository.rb [hunk 1]
```diff
@@ -81,126 +79,115 @@ def worst_coach(season)
   end
 
 
-def most_tackles(season_id)
-  games_in_the_season = []
-  tackles_team = {}
-  @game_collection.each do |game|
-    if game.season == season_id
-      games_in_the_season << game.game_id
+  def most_tackles(season_id)
```
Comment (janegreene) [inline lib/season_repository.rb:82]: fixed indention

File: lib/season_repository.rb [hunk 2]
```diff
@@ -81,126 +79,115 @@ def worst_coach(season)
...
     end
-  end
-  games_in_the_season.each do |season_game|
-    @game_team_collection.each do |game_team|
-      if game_team.game_id == season_game
-        if tackles_team.key?(game_team.team_id)
-          tackles_team[game_team.team_id] += game_team.tackles
-        else
-          tackles_team[game_team.team_id] = game_team.tackles
+    games_in_the_season.each do |season_game|
```
Comment (janegreene) [inline lib/season_repository.rb:90]: fixed indention

File: lib/season_repository.rb [hunk 3]
```diff
@@ -81,126 +79,115 @@ def worst_coach(season)
...
+          else
+            tackles_team[game_team.team_id] = game_team.tackles
+          end
         end
       end
     end
+    team_id_to_find = tackles_team.sort_by { |key, value| value }.first[0]
+    find_team_id(team_id_to_find)
   end
-  team_id_to_find = tackles_team.sort_by { |key, value| value }.first[0]
```
Comment (janegreene) [inline lib/season_repository.rb:126]: see fixed indention above

File: lib/season_repository.rb [hunk 4]
```diff
@@ -81,126 +79,115 @@ def worst_coach(season)
...
-  find_team_id(team_id_to_find)
-end
-
-def find_team_id(id)
-  found_team = @team_collection.find do |team|
-    team.team_id == id
-  end
-  named_team = found_team.teamname
-  named_team
-end
```
Comment (janegreene) [inline lib/season_repository.rb:136]: removed for use in module

File: lib/findable.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+module Findable
+
```
Comment (EricLarson2020) [inline lib/findable.rb:2]: Nice work good use of a module

**Review by EricLarson2020 — APPROVED** [review]
Nice job here

**Pull request merged**